### ISSUE TITLE
Add ppc64le support for RH 8 clones

### ISSUE THREAD

**Issue by raulillo82** [uyuni-project/uyuni#5833]
# Short description

Add ppc64le support for RH 8 clones

# Details

There is a SUSE Manager customer that would like to manage ppc64le clients running RH 8 within their SUSE Manager. Seeing the lack of support for that, Uyuni would be a good starting point. Moreover, whatever ends up in "spacewalk-common-channels" can be easily integrated into SUSE Manager as well, even if there may only be L1 support.

Customer would like to specifically use the salt bundle. Apparently, AlmaLinux 8.6 has support for ppc64le, so maybe that could be a good starting point for getting the Salt bundle from there.

Once this is done (if it is done and if it can be done), that would also be a starting point for the future support of RH 9 clones, so that this architecture can also be added.

Let me know if I can help in any way.
I saw an old PR: #2738. However this might not apply any more due to the fact of centos being out of scope as of today.

**Comment by raulillo82** [comment]
Can please anyone look into this?

**Comment by juliogonzalez** [comment]
The instructions at https://www.uyuni-project.org/uyuni-docs/en/uyuni/client-configuration/registration-overview-redhat.html should work, but something tells me they are not fully correct for RHEL, as they reference RES client tools, that are available for SUSE customers, and not every Uyuni user is a SUSE customer.

Maybe you can help preparing a fix for this.

I'd follow the instructions for RHUI on an AWS Account, **without** adding any SCC credentials to the server.

Things should work, but adding the client tools should not. At that point I think you will need to patch `spacewalk-common-channels.ini` to include info there for RHEL.

I am not sure if patching the bootstrap repo definitons will be needed.

Once that works, a PR can be created for this repository, and then another one will be needed for the docs, keeping the docs compatible with both SUSE Manager and Uyuni.

**Comment by nodeg** [comment]
@juliogonzalez Do you think we still need this?

**Comment by juliogonzalez** [comment]
No clue if there's still interest or not.

Question for @admd for SUSE Multi-Linux Manager, and for the community for Uyuni. Adding it for Uyuni should be trivial for 8, 9 and 10.
